tests/fixtures/plugin/build.atpkg

```diff
@@ -0,0 +1,29 @@
+;; Copyright (c) 2016 Anarchy Tools Contributors.
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;   http:;;www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+
+(package
+  :name "plugin"
+  
+  :tasks {
+    :a {:tool "nop" }
+    :default {
+      :tool "echo.plugin"
```

Comment:
I think requiring ".plugin" is bit against what you're going for: `atbuild` being a light-weight thing just made up of a bunch of potential tools. For example, why are the bundled/built-ins considered more important?